--- a/web3/utils/normalizers.py	2019-08-23 14:39:41.263079574 +0300
+++ b/web3/utils/normalizers.py	2019-08-23 14:40:04.809324547 +0300
@@ -144,10 +144,9 @@
                 "Could not look up name %r because ENS is"
                 " set to None" % (val)
             )
-        elif int(w3.net.version) is not 1 and not isinstance(w3.ens, StaticENS):
+        elif isinstance(w3.ens, StaticENS):
             raise InvalidAddress(
-                "Could not look up name %r because web3 is"
-                " not connected to mainnet" % (val)
+                "Could not look up name %r because web3 ens is StaticENS" % (val)
             )
         else:
             return (abi_type, validate_name_has_address(w3.ens, val))
