--- orig/CMakeLists.txt  2018-03-10 17:46:49.605769576 +0300
+++ src/CMakeLists.txt	 2018-03-10 17:47:17.178781679 +0300
@@ -55,7 +55,7 @@
             # generate pkg-config file
             configure_file("${BASENAME}.in" "${BASENAME}" @ONLY)
             install(FILES       "${CMAKE_CURRENT_BINARY_DIR}/${BASENAME}"
-                    DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+                    DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
         endif()
     endif()
 endfunction(generate_pkgconfig)
